areas/extension/src/AzureMcp.Extension/Commands/AzCommand.cs

```diff
@@ -1,47 +1,24 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
Let's make sure our manual test team runs through all the scenario tests to make sure this works as expected.